.travis_scripts/openapi-validator.sh

```diff
@@ -2,13 +2,23 @@
 set -eu
 set -o pipefail
 
-if ! ./openapi-generator-cli version
+ver=4.3.1
+if ./openapi-generator-cli version
```

Comment:
@bzwei This line is going to do the download.
Why are we not capturing the output of the command 
```
VERSION=`./openapi-generator-cli version`
```
And using that along with
```
OPENAPI_GENERATOR_VERSION=${VERSION} ./openapi-generator-cli validate -i "$entry"
```
I had mentioned this here https://github.com/RedHatInsights/approval-api/pull/358#discussion_r427584395

Reply:
@mkanoor I was thinking parsing from the jar filename to get version is more accurate in case the version command may produce another format other than pure version numbers in the future. 
Anyway I now switch to directly capture output.